__tests__/app.test.js

```diff
@@ -296,3 +296,22 @@ describe("POST: /api/articles/:article_id/comments", () => {
         .expect(404)
     });
 })
+
+describe("DELETE /api/comments/:comment_id", () => {
+    test("200: Successfully deletes comment and returns deleted comment", () => {
+        return request(app)
+        .delete('/api/comments/1')
+        .expect(200)
+        .then((res) => {
+            expect(res.body.comment).toMatchObject({
+                comment_id: 1,
+                body: "Oh, I've got compassion running out of my nose, pal! I'm the Sultan of Sentiment!",
+                article_id: 9,
+                author: "butter_bridge",
+                votes: 16,
+                created_at: "2020-04-06T12:17:00.000Z"
+            })
+        })
+        
+    })
```

Comment:
Should the status code for a successful delete be 200? I believe there is a more appropriate status code for this.